src/components/header/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { themeChange } from "theme-change";
+import { useEffect, useState } from "react";
+import Link from "next/link";
```

Comment:
Could you kindly ensure that there is a space between the various import types, please?